config/cmake/addGitHooks.cmake

```diff
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 2.8.7)
+
+find_package(Git)
+if(NOT GIT_FOUND)
+    message(FATAL_ERROR "git not found!")
+endif()
+
+include(clang-cxx-dev-tools)
```

Comment:
This may already be included earlier @ https://github.com/GMLC-TDC/HELICS/blob/master/CMakeLists.txt#L694

```suggestion
```

Reply:
We probably want a minimum version of clang-format.  An old one might not work properly.

Reply:
I don't think there are any issues linking to clang-format if a different compiler is used.